rpc/block_svc_routines.c

```diff
@@ -3871,8 +3894,9 @@ block_create_cli_1_svc_st(blockCreateCli *blk, struct svc_req *rqstp)
 
   LOG("mgmt", GB_LOG_INFO,
       "create cli request, volume=%s blockname=%s mpath=%d blockhosts=%s "
-      "authmode=%d size=%lu, rbsize=%d", blk->volume, blk->block_name, blk->mpath,
-      blk->block_hosts, blk->auth_mode, blk->size, blk->rb_size);
+      "authmode=%d size=%lu, rbsize=%d, blksize: %d", blk->volume,
```

Comment:
Just to align with others, should this be blksize=%d ?

Reply:
yeah, will fix it.